core/object/class_db.h

```diff
@@ -102,12 +102,12 @@ class ClassDB {
 
 public:
 	struct PropertySetGet {
-		int index;
+		int index = 0;
 		StringName setter;
 		StringName getter;
-		MethodBind *_setptr;
-		MethodBind *_getptr;
-		Variant::Type type;
+		MethodBind *_setptr = nullptr;
+		MethodBind *_getptr = nullptr;
+		Variant::Type type = Variant::NIL;
```

Comment:
`clang-tidy` didn't change this but I did it manually - as far as I know we should also initialize enum values, otherwise it would also contain garbage data which is not from the enum values.